src/services/pool/types.ts

```diff
@@ -1,48 +1,10 @@
 import { BigNumber } from 'ethers';
-
+import { Pool as SDKPool, PoolToken } from '@balancer-labs/sdk';
 import { Address } from '@/types';
 
-export interface Pool {
-  id: string;
-  address: string;
-  poolType: PoolType;
-  swapFee: string;
-  owner: string;
-  factory: string;
-  tokens: PoolToken[];
-  tokensList: string[];
-  totalLiquidity: string;
-  totalShares: string;
-  totalSwapFee: string;
-  totalSwapVolume: string;
-  createTime: number;
+export interface Pool extends SDKPool {
   onchain?: OnchainPoolData;
-  mainTokens?: string[];
-  wrappedTokens?: string[];
   linearPoolTokensMap?: Record<string, PoolToken>;
-  unwrappedTokens?: string[];
-  isNew?: boolean;
-  volumeSnapshot?: string;
-  feesSnapshot?: string;
-  apr?: PoolAPRs;
-  boost?: string;
-}
-
-export enum PoolType {
-  Weighted = 'Weighted',
-  Investment = 'Investment',
-  Stable = 'Stable',
-  MetaStable = 'MetaStable',
-  StablePhantom = 'StablePhantom',
-  LiquidityBootstrapping = 'LiquidityBootstrapping',
-}
```

Comment:
This is used in a lot of places, why is removed?

Reply:
Grabbing it from SDK instead.

Reply:
Where though? The SDK doesn't provide the same PoolType and there are ts errors for this in multiple places. See my fixes PR: https://github.com/balancer-labs/frontend-v2/pull/2208